Description: Adds keywords to desktop file to prevent Lintian
complaint about desktop-entry-lacks-keywords-entry
Forwarded: no
Author: Daniel Stender <debian@danielstender.com>

--- a/data/misc/gummi.desktop.in
+++ b/data/misc/gummi.desktop.in
@@ -8,5 +8,6 @@
 Terminal=false
 Type=Application
 Categories=Office;
+Keywords=latex;editor;
 StartupNotify=true
 MimeType=text/x-tex;
